File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.java
class Solution {
    public int getMinDistance(int[] nums, int target, int start) {
        
        for (int inc = start, dec = start - 1; dec >= 0 || inc < nums.length; dec--, inc++) {
            if (inc < nums.length) {
                if (nums[inc] == target) {
                    return Math.abs(inc - start);
                }
            }

            if (dec >= 0) {
                if (nums[dec] == target) {
                    return Math.abs(start - dec);
                }
            }
        }
        
        return start;
    }
}

File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.py
class Solution:
    def getMinDistance(self, nums: List[int], target: int, start: int) -> int:
        inc = start 
        dec = start - 1

        while dec >= 0 or inc < len(nums):
            if inc < len(nums) and nums[inc] == target:
                return abs(inc - start)
        
            if dec >= 0 and nums[dec] == target:
                return abs(start - dec)

            inc += 1
            dec -= 1

        
        return start